src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,233 @@
+import React, {Component} from 'react';
+import {block} from '../utils/cn';
+import _ from 'lodash';
+
+import './Progress.scss';
+
+const b = block('progress');
+
+export type Theme = 'default' | 'success' | 'warning' | 'danger' | 'info' | 'misc';
+export type View = 'normal' | 'thin' | 'thinnest';
+export type Value = number;
```

Comment:
These name are too generic to be exported from uikit, `ProgressTheme`, `ProgressView`, `ProgressValue` are better